nmdc_automation/re_iding/base.py

```diff
@@ -51,6 +51,19 @@ def __init__(self, api_client: NmdcRuntimeApi, data_dir: str,
             template_file = NAPA_TEMPLATE
         with open(template_file, "r") as f:
             self.workflow_template = yaml.safe_load(f)["Workflows"]
+        # collector to track ID changes as (type, old_id, new_id)
+        self.id_changes = []
+
+    def write_id_changes(self, study_id: str):
+        """
+        Write ID changes to a file.
+        """
+        filename = f"{study_id}_id_changes.tsv"
```

Comment:
I am wondering whether there is a chance that the `study_id` string contain characters that are not allowed to exist in filenames (e.g. `/`). Here's a link to a [Stack Overflow post about the topic](https://stackoverflow.com/a/31976060).

Reply:
For our purposes, no. For applying the current reID changes, we know all of the study_ids ahead of time. The way the current ETL pipeline is set up, there is an assumption that things like `study_id` have already been updated and conform to our schema pattern